### ISSUE TITLE
Add Font Awesome icons

### ISSUE THREAD

**Issue by gEndelf** [evolus/pencil#115]
* Operating system: ALL
* Pencil version: ALL

It would be nice to have predefined icons from public icon toolkits (font awesome/others)

**Comment by esimotas** [comment]
It is easy enough to generate a stencil from the svg icons.
Here is a link to a [fontawsome stencil](https://drive.google.com/open?id=0B_e7A_rmJnVQWHlqaGJhR1E3d1E)

**Comment by HEXcube** [comment]
The one at [Google Code repo](https://code.google.com/archive/p/evoluspencil/downloads) is outdated. This Google Groups post however, does contain two recently generated Font Awesome stencils as attachments: [See Link](https://groups.google.com/forum/#!topic/pencil-user/Pw0CtomE3Os)

**Comment by Boimb** [comment]
Hi,
I must miss something... Can't install any of those package (zip or via xml), neither in pensil v2 nor in pencil v3beta.
Using Mac version.
Any tips ?

**Comment by HEXcube** [comment]
> Ok, it's working if installing in the "Collections" tab not in "My Stuff" nor via "Load Dev Stencil directory"

Yeah, that's how u install stencil zips in Pencil 3.x. But, I wish the new Font Awesome was listed there on the Collections menu, so that we don't have to go hunting stencil zips. :grin:

**Comment by thekalinga** [comment]
You can use the following generator for the stencils

https://github.com/1tontech/pencil-stencil-generator

**Comment by sTiKyt** [comment]
@gEndelf this repository seems outdated, please post your Issue to our fork! We'll updаte it soon!
https://github.com/DarkSecDevelopers/RePencil
